doc/Extensions/Services.md

```diff
@@ -1,12 +1,144 @@
 source: Extensions/Services.md
 path: blob/master/doc/
-# Setting up Services
+[TOC]
+
+# Setting up services
 
 Services within LibreNMS provides the ability to use Nagios plugins to perform additional monitoring outside of SNMP.
 
 **These services are tied into an existing device so you need at least one device to be able to add it
 to LibreNMS - localhost is a good one. This is needed in order for alerting to work properly.**
 
+## Pre installed plugins
+
+List of pre installed plugins in the format: \<name\> - \<origin\>
```

Comment:
Quick comment: They dont come pre installed. 

Afaik those plugins comes with the nagios-plugins/nagios-plugins-all packages for Ubuntu and CentOS respectively.

Reply:
I cannot find such package under Ubuntu repos however their does appear to be one for centos.

>To avoid confusion, I'll swap pre installed with supported or anything else to dont let the user think they come already with the LibreNMS install.

Yes that is better wording.

However with the VM images they are pre packaged.

Reply:
In many distros the package is called monitoring-plugins